thirdeye/thirdeye-frontend/app/pods/components/self-serve-alert-details/component.js

```diff
@@ -38,6 +39,7 @@ export default Component.extend({
     }
   },
 
+  // eslint-disable-next-line ember/avoid-leaking-state-in-ember-objects
```

Comment:
Lets define the object outside the component as a const to avoid bypassing the eslint.